services/EventTokenService.js

```diff
@@ -0,0 +1,66 @@
+const jwt = require("jsonwebtoken");
+const crypto = require("crypto");
+
+// A service class for Token generation and management
+
+class EventTokenService {
+  static #app_access_key = config.get("Event100ms.APP_ACCESS_KEY");
+  static #app_secret = config.get("Event100ms.APP_SECRET");
```

Comment:
Since we are using env here, we should set them in default and CUSTOM_ENVIRNMANE_VARIABLES files in /config

Reply:
Yes, added it. Thank you.